pages/api/members/index.js

```diff
@@ -7,10 +7,12 @@ const BYPASS_AUTH_ORIGINS = ['https://teamwaterloop.ca'];
 module.exports = async (req, res) => {
     await data.initIfNotStarted();
     if (req.method === 'POST') {
+        const authStatus = true;
```

Comment:
This is a nitpick, but could you actually set up the code so that `authStatus = false` initially? The functionality is the same, but in case the conditional block fails to run for some reason it's not secure to have `authStatus` true by default. Something like this (if you choose to copy-paste, please fix formatting before pushing again):

```
const authStatus = false;

if(BYPASS_AUTH_ORIGINS.includes(req.headers['origin'])) {
            authStatus = true;
} else {
            const token = cookie.parse(req.headers.cookie).token;
            authStatus = await data.auth.checkAnyUser(`Bearer ${token}`, res);
}
```

Reply:
Yeah, this is actually what I was thinking as well since, in the previous code, authStatus isn't set to true by default, it could be false if the OR of the two conditions is false.